lib/jazzy/grouper.rb

```diff
@@ -49,22 +49,29 @@ def self.group_docs_by_module(docs, type_category_prefix)
 
     def self.group_custom_categories(docs, doc_index)
       group = config.custom_categories.map do |category|
-        children = category['children'].map do |name|
-          unless doc = doc_index.lookup(name)
-            warn 'WARNING: No documented top-level declarations match ' \
-              "name \"#{name}\" specified in categories file"
-            next nil
+        children = category['children'].map do |selector|
+          selected = if selector.is_a?(String)
+            unless doc = doc_index.lookup(selector)
+              warn 'WARNING: No documented top-level declarations match ' \
+                "name \"#{name}\" specified in categories file"
```

Comment:
`name` is wrong here, cut + paste error - is causing the swift_spec test to fail